### DISCUSSION TITLE
So what exactly happens when I enable say, rndis0 or wlan1 after enableing usb/wifi hotspot?

### DISCUSSION THREAD

**Discussion by Virkis** [Mygod/VPNHotspot#278] [General]
More of a curiosity, really. I've used this app for two or so years now and always enable(?) them after turning on whatever tether method, but never really understood why. It feels like something I should do, but I never noticed a difference in performance with either one on or off. I first found this app to use a VPN through it, but that was only for a few months and was years ago now. 

Does it matter? Is it a VPN only thing? Or is it enabling something behind the scenes that make networking across connected devices easier? I've looked every time I've checked this git page for an update and nobody seems to have mentioned it. 

So.. Yeah. What does the button do?

**Comment by Mygod** [comment]
USB/Wi-Fi hotspot switches are only shortcuts to system tethering. The app does not (really) work unless you switch on the corresponding interfaces afterwards.
